controllers/v1/cloud-services/gcp.js

```diff
@@ -180,5 +180,96 @@ module.exports = class Gcp {
     })
    }
 
+
+       /**
+     * @api {post} /kendra/api/v1/cloud-services/gcp/uploadFile  
+     * upload file to presignedUrl
+     * @apiVersion 1.0.0
+     * @apiGroup Gcp
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiParamExample {fromData} Request:
+     * {
+     * "filePath" : "1230981723091723/sample.csv"
+     * "bucketName":"sl-dev-storage",
+     * "storage": "GC"
+     *  
+     * }
+     * @apiSampleRequest /kendra/api/v1/cloud-services/gcp/uploadFile
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+     * "status": 200,
+     * "message":"File uploaded successfully",
+     *  "result": {
+     *   "kind": "storage#object",
+     *   "id": "sl-dev-storage/my.csv/1590132715837085",
+     *   "selfLink": "https://www.googleapis.com/storage/v1/b/sl-dev-storage/o/my.csv",
+     *   "mediaLink": "https://storage.googleapis.com/download/storage/v1/b/sl-dev-storage/o/my.csv?generation=1590132715837085&alt=media",
+     *   "name": "my.csv",
+     *   "bucket": "sl-dev-storage",
+     *   "generation": "1590132715837085",
+     *   "metageneration": "1",
+     *   "contentType": "text/csv; charset=utf-8",
+     *   "storageClass": "REGIONAL",
+     *   "size": "137",
+     *   "md5Hash": "edPW+5pdDV6cbSl1TkVwLA==",
+     *   "contentEncoding": "gzip",
+     *   "crc32c": "56+5+A==",
+     *   "etag": "CJ2tsf35xukCEAE=",
+     *   "timeCreated": "2020-05-22T07:31:55.836Z",
+     *   "updated": "2020-05-22T07:31:55.836Z",
+     *   "timeStorageClassUpdated": "2020-05-22T07:31:55.836Z"
+     *    }
+     * }
+     */
+
+    /**
+      * To upload file cloud 
```

Comment:
@rakeshSgr - Change name to - Upload file to GCP

Reply:
resolved